src/components/UI/Text/InputNumber.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="grid grid-cols-1">
-    <div class="flex justify-between">
+    <div class="flex justify-between" v-if="isTitlePresent || isValidationEnabled">
```

Comment:
Maybe because of this change, the validation message is not showing up at all. Whatever you type, the message "mobile number is invalid" doesn't show up